### PR TITLE
add the title for patch

### PR DESCRIPTION
**Pull request by bayern39**
the issue #31 has been resolved ,add the title for the patch

### CODE REVIEW

File: src/main/java/org/jboss/set/aphrodite/domain/Patch.java [hunk 1]
```diff
@@ -30,20 +30,25 @@
...
-    public Patch(String id, URL url, Repository repository, Codebase codebase, PatchStatus status, String description) {
+    public Patch(String id, URL url, Repository repository, Codebase codebase, PatchStatus status, String title, String body) {
         this.id = id;
         this.url = url;
         this.codebase = codebase;
         this.status = status;
-        this.description = description;
+        this.title = title;
+        this.body = body;
+        this.description = body + title;
```
Comment (elguardian) [inline src/main/java/org/jboss/set/aphrodite/domain/Patch.java]: this is a derived atrribute. not need to hold the info.
remove the  attribute. description

File: src/main/java/org/jboss/set/aphrodite/domain/Patch.java [hunk 2]
```diff
@@ -66,6 +72,23 @@ public void setStatus(PatchStatus status) {
...
+    public String getBody() {
+        return body;
+    }
+
+    public void setBody(String body) {
+        this.body = body;
+    }
+
     public String getDescription() {
         return description;
```
Comment (elguardian) [inline src/main/java/org/jboss/set/aphrodite/domain/Patch.java]: you can do here return  title + body

File: src/main/java/org/jboss/set/aphrodite/domain/Patch.java [hunk 3]
```diff
@@ -87,6 +110,8 @@ public String toString() {
         return "Patch{" +
                 "url=" + url +
                 ", status=" + status +
+                ", title='" + title + '\'' +
+                ", body='" + body + '\'' +
                 ", description='" + description + '\'' +
```
Comment (elguardian) [inline src/main/java/org/jboss/set/aphrodite/domain/Patch.java]: this is redundant here. you are going to print the same twice.

**Comment by bayern39** [comment]
ok you are right,i got it ,Thx

**Pull request closed without merging**